test/db/formats/mach0/mach0

```diff
@@ -267,8 +267,8 @@ CMDS=ik ***~uuid
 EXPECT=<<EOF
 info/mach0_uuid_command.format=[4]Ed[16]b (mach0_load_command_type)cmd cmdsize uuid
 info/mach0_cmd_8.format=mach0_uuid_command
-info/uuid.0=ff4d6c2e1dc0369d98d0d3040a8049e8
 info/mach0_cmd_8.cmd=uuid
+info/uuid.0=ff4d6c2e1dc0369d98d0d3040a8049e8
```

Comment:
So this is technically a change in functionality, but I think it makes sense (`info/mach0_cmd_8` lines are grouped together now).
The change is probably due to the manual memory management for `libs` that is now replaced by a `r_pvector`.